Makefile

```diff
@@ -0,0 +1,28 @@
+all: test
+
+PACKAGES := $(shell go list ./... | grep -v vendor)
+
+init:
+	go get -u github.com/AlekSi/gocoverutil
+	go get -u gopkg.in/alecthomas/gometalinter.v1
+	gometalinter.v1 --install
+
+install:
+	go install -v $(PACKAGES)
+	go test -v -i $(PACKAGES)
+
+install-race:
+	go install -v -race $(PACKAGES)
+	go test -v -race -i $(PACKAGES)
```

Comment:
Hm, but I think `go install -race` requires writeable GOROOT, that's ok on macOS, but may not work on Linux, it depends on how Go was installed. @percona-csalguero, does it work in your dev environment?